paper/src/main/java/com/untamedears/jukealert/gui/SnitchOverviewGUI.java

```diff
@@ -30,7 +30,7 @@ public SnitchOverviewGUI(Player player, List<Snitch> snitches, String title, boo
 	}
 
 	private List<IClickable> constructSnitchClickables() {
-		final List<IClickable> clickables = new LinkedList<>();
+		final List<IClickable> clickables = new ArrayList<>();
```

Comment:
```suggestion
		final List<IClickable> clickables = new ArrayList<>(this.snitches.size());
```

Reply:
I can't believe I missed this, thank you ❤️